ietf/utils/test_runner.py

```diff
@@ -566,6 +566,14 @@ def ignore_pattern(regex, pattern):
                         covered.add(regex)
                         break
 
+            uncovered = set()
+            for regex, compiled, obj in patterns:
+                if regex not in covered:
+                    uncovered.add(regex)
+
+            if(len(uncovered) > 0):
+                print("Uncovered URLs", uncovered)
```

Comment:
Did you mean to leave this in the commit? Maybe have it write to a file instead of stdout?

Reply:
Yes, just to see some debug, although I will remove it again soon

Reply:
Removed.